src/test/java/com/celements/course/service/CourseServiceTest.java

```diff
@@ -355,11 +364,25 @@ private void expectEmail(String email, int count) throws Exception {
   }
 
   @Test
-  public void test_getConfirmState_confirmed() throws Exception {
+  public void test_getConfirmState_celdev581UseCase1() throws Exception {
```

Comment:
use readable naming, perhaps add javadoc to explain scenario if name is insufficient